Parsers/ASimNetworkSession/ProductParsers/vimNetworkSessionMicrosoft365Defender.yaml

```diff
@@ -141,7 +141,7 @@ ParserQuery: |
        SrcDomain = UrlDomain,
        SrcFQDN = RemoteUrl
     | extend  
-       DstHostname = DvcHostname,
+       DstHostname = tostring(DvcHostname),
```

Comment:
Again, did you see an example of DvcHostname not being a string?

Reply:
yup, dynamic type